esphome/components/tm1637/tm1637.cpp

```diff
@@ -131,6 +131,8 @@ void TM1637Display::setup() {
 void TM1637Display::dump_config() {
   ESP_LOGCONFIG(TAG, "TM1637:");
   ESP_LOGCONFIG(TAG, "  INTENSITY: %d", this->intensity_);
+  ESP_LOGCONFIG(TAG, "  INVERTED: %d", this->inverted_);
+  ESP_LOGCONFIG(TAG, "  LENGTH: %d", this->length_);
```

Comment:
Not sure why the one that was there is all-caps, but we generally don't do that:
```suggestion
  ESP_LOGCONFIG(TAG, "  Intensity: %d", this->intensity_);
  ESP_LOGCONFIG(TAG, "  Inverted: %d", this->inverted_);
  ESP_LOGCONFIG(TAG, "  Length: %d", this->length_);
```